src/assets/modifierdata/ranger.yaml

```
@@ -1,456 +1,511 @@
+      subText: no sic em, sharpened edges
+      amountData:
+        label: 'hit/s'
+        default: 0
+        quantityEntered: 1
+      modifiers:
+        attributes:
+          Power Coefficient: 435
+        conversionAfterBuffs:
+          Bleeding Coefficient: {Critical Chance: 100%}
+      gw2id: 45717
+
+- section: Marksmanship
+  id: 8
+  items:
+
+    - id: farsighted
+      text: Farsighted
+      subText: melee
+      modifiers:
+        damage:
+          Strike Damage: [5%, mult]
+      gw2id: 1000
+      defaultEnabled: true
+
+    - id: farsighted-2
+      text: Farsighted
+      subText: ranged
+      modifiers:
+        damage:
+          Strike Damage: [15%, mult]
+      gw2id: 1000
+
+    - id: predators-onslaught
+      text: Predator's Onslaught
+      amountData:
+        label: '% impaired'
+        default: 100
+        quantityEntered: 100
+      modifiers:
+        damage:
+          Strike Damage: [15%, mult]
+      gw2id: 996
+      defaultEnabled: true
+
+- section: Skirmishing
+  id: 30
+  items:
+
+    - id: sharpened-edges
+      text: Sharpened Edges
+      subText: per hit (including non critical)
+      amountData:
+        label: 'hit/s'
+        default: 0
+        quantityEntered: 1
+      modifiers:
+        conversionAfterBuffs:
+          Bleeding Coefficient: {Critical Chance: 100%}
+      gw2id: 1069
+      defaultEnabled: false
+
+    - id: hidden-barbs
+      text: Hidden Barbs
+      modifiers:
+        damage:
+          Bleeding Damage: [33%, unknown]
+      gw2id: 1846
+      defaultEnabled: true
+
+    - id: hunters-tactics
+      text: Hunter's Tactics
+      subText: 100% flanking
+      modifiers:
+        damage:
+          Strike Damage: [10%, mult]
+        attributes:
+          Critical Chance: 10%
+      minor: true
+      gw2id: 1068
+      defaultEnabled: true
+
+    - id: light-on-your-feet
+      text: Light on your Feet
+      subText: 100% uptime
+      modifiers:
+        damage:
+          Strike Damage: [10%, mult]
+        attributes:
+          Condition Duration: 10%
+      gw2id: 1912
+
+    - id: vicious-quarry
+      text: Vicious Quarry
+      subText: 100% fury
+      modifiers:
+        attributes:
+          Ferocity: [250, buff]
+          Critical Chance: 10%
+      gw2id: 1888
+      defaultEnabled: true
+
+- section: Wilderness Survival
+  id: 33
+  items:
+
+    - id: oakheart-salve
+      text: Oakheart Salve
+      subText: 100% regeneration
+      modifiers:
+        damage:
+          Damage Reduction: [5%, unknown]
+      gw2id: 1086
+      defaultEnabled: true
+
+    - id: taste-for-danger
+      text: Taste for Danger
+      modifiers:
+        attributes:
+          Vitality: [120, buff]
+          Expertise: [8, buff] # conversion of own 120 vitality
+        conversion:
+          Expertise: {Vitality: 7%}
+      wvwModifiers:
+        attributes:
+          Vitality: [120, buff]
+          Expertise: [8, buff] # conversion of own 120 vitality

```

```suggestion
          Expertise: [5, buff] # conversion of own 120 vitality
```
I assume the bonus expertise in WvW mode would be 0.04 * 120 = 5 instead of 8 here!